modules/ocs2/src/main/scala/gem/ocs2/pio/PioDecoder.scala

```diff
@@ -23,16 +24,16 @@ object PioDecoder {
         f(n)
     }
 
-  def enum[A](dataType: String)(m: (String, A)*): PioDecoder[A] =
-    fromParseFunction[A](dataType) { m.toMap.lift }
+  def enum[A: ClassTag](m: (String, A)*): PioDecoder[A] =
```

Comment:
I tend to forget about scala-js.  Thanks for pointing this out.  It might have been okay in this case because nothing here is intended to be used from a client app but still this is better.